tests/plugins/test_plugin.py

```diff
@@ -136,7 +130,6 @@ class AirflowTestPlugin(AirflowPlugin):
     name = "test_plugin"
     operators = [PluginOperator]
     sensors = [PluginSensorOperator]
-    hooks = [PluginHook]
```

Comment:
Should we also remove operators/sensors above?

Reply:
Yup that was removed in https://github.com/apache/airflow/pull/43290